ctapipe/tools/tests/test_tools.py

```diff
@@ -131,7 +131,7 @@ def _generator(self):
         f.write(b"dummy")
         f.flush()
 
-    config = files("ctapipe.examples").joinpath("stage1_config.json")
+    config = files("ctapipe.tools.tests.resources").joinpath("stage1_config.json")
```

Comment:
Do we have `zip_safe=False` in setup.py? Otherwise you should use https://docs.python.org/3/library/importlib.html#importlib.resources.as_file



```
resources = files("ctapipe.tools.tests.resources")
with as_file(resources.joinpath("stage1_config.json")) as config_path:
    ...
```

Reply:
we do: https://github.com/cta-observatory/ctapipe/blob/15cd535e1025352de8477240cd3f8b5da1038147/setup.py#L82